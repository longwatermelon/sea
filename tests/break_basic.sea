let write_buf:int = 0;

fn write_char(c:int) -> void {
    write_buf = c;
    syscall(4, 1, &write_buf, 1);
}

fn main() -> int {
    let i:int=0;
    while (i<10) {
        if (i==5) {
            break;
        }
        write_char(i+48);
        i=i+1;
    }
    return 0;
}
