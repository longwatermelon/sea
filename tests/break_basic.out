RETURN_CODE:0
STDOUT:01234
